docs/src/testing/debugging.md

```diff
@@ -71,11 +71,6 @@ just `original value` is printed.
 > - Make sure to remove all of the prints before compiling the final version of your contract. 
 > - This feature is highly experimental and breaking changes are to be expected.
 
-To allow printing in tests you have to add the following line to your `Scarb.toml`:
-```toml
-[[target.starknet-contract]]
-allowed-libfuncs-deny = false
-```
```

Comment:
Unrelated, but it was just not true, it is the default setting for starknet target in Scarb